modules/core/src/main/scala/scala/build/internals/EnvVar.scala

```diff
@@ -124,6 +153,40 @@ object EnvVar {
     val sparkHome = EnvVar("SPARK_HOME", "Spark installation directory", requiresPower = true)
   }
 
+  object Bloop extends EnvVarGroup {
+    override def groupName: String = "Bloop"
+    override def all = Seq(
+      bloopComputationCores,
+      bloopDaemonDir,
+      bloopExportJarClassifiers,
+      bloopJavaOpts,
+      bloopModule,
+      bloopPort,
+      bloopScalaVersion,
+      bloopVersion,
+      bloopServer,
+      bloopExtraTimeout,
+      bloopTravis
+    )
+    val bloopComputationCores = EnvVar(
+      "BLOOP_COMPUTATION_CORES",
+      "Number of computation cores to be used",
+      requiresPower = true
+    )
+    val bloopDaemonDir = EnvVar("BLOOP_DAEMON_DIR", "Bloop daemon directory", requiresPower = true)
+    val bloopExportJarClassifiers =
+      EnvVar("BLOOP_EXPORT_JAR_CLASSIFIERS", "Export jar classifiers", requiresPower = true)
```

Comment:
That's for sbt only, not sure how useful here

Reply:
removed this, too